LICENSE-binary

```diff
@@ -278,6 +277,7 @@ BSD 3-Clause
 org.antlr:antlr-runtime:4.8-1
 org.ow2.asm:asm:5.0.4
 org.jline:jline:3.21.0
+com.github.luben.zstd-jni:1.3.2-6
```

Comment:
Hi, I checked https://github.com/luben/zstd-jni#license and find out this package is actually under BSD 2-Clause license. 

https://github.com/luben/zstd-jni#license

Reply:
OK, I see and changed.